ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/minimal.yaml

```diff
@@ -12,7 +12,7 @@ CONFIG_NAME: 'minimal'
 
 # Transition
 # ---------------------------------------------------------------
-# TBD, 2**256-2**10 is a placeholder
+# 2**256-2**10 for fake minimal network
```

Comment:
not sure i'd say 'fake', maybe 'testing' or something?

Reply:
It's straight from the spec. :)